appcues/src/main/java/com/appcues/ui/ExperienceRenderer.kt

```diff
@@ -107,15 +108,20 @@ internal class ExperienceRenderer(
             return true
         }
 
-        val success = show(qualifiedExperiences.first())
-        if (!success) {
+        qualifiedExperiences.filter { it.renderContext is RenderContext.Embed }.forEach {
+            // TODO what to do here
+            show(it)
+        }
```

Comment:
using iOS as a guide, this is where the basic caching stuff comes into play
* keep some set of potentially renderable experiences https://github.com/appcues/appcues-ios-sdk/blob/main/Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceRenderer.swift#L76
* try and show things right away, if possible https://github.com/appcues/appcues-ios-sdk/blob/main/Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceRenderer.swift#L78-L80
* remove things from cache when an experience is dismissed/completed https://github.com/appcues/appcues-ios-sdk/blob/main/Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceRenderer.swift#L232

The full set of rules is written up here for reference https://github.com/appcues/appcues-mobile-experience-spec/blob/main/rfc/embedded-experiences.md#processing-embeds-in-qualification-response

I think these behaviors can be layered on after this PR though as a next step, no issue with that.

Reply:
yes, seems totally fine to me - makes sense to layer that cache behavior on top in the next round of work.